src/main/java/org/alliancegenome/curation_api/services/validation/dto/fms/CrossReferenceFmsDTOValidator.java

```diff
@@ -0,0 +1,70 @@
+package org.alliancegenome.curation_api.services.validation.dto.fms;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.alliancegenome.curation_api.constants.ValidationConstants;
+import org.alliancegenome.curation_api.model.entities.CrossReference;
+import org.alliancegenome.curation_api.model.entities.ResourceDescriptor;
+import org.alliancegenome.curation_api.model.entities.ResourceDescriptorPage;
+import org.alliancegenome.curation_api.model.ingest.dto.fms.CrossReferenceFmsDTO;
+import org.alliancegenome.curation_api.response.ObjectResponse;
+import org.alliancegenome.curation_api.services.ResourceDescriptorPageService;
+import org.alliancegenome.curation_api.services.ResourceDescriptorService;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.inject.Inject;
+
+@RequestScoped
+public class CrossReferenceFmsDTOValidator {
+
+	@Inject ResourceDescriptorService resourceDescriptorService;
+	@Inject ResourceDescriptorPageService resourceDescriptorPageService;
+
+	public ObjectResponse<List<CrossReference>> validateCrossReferenceFmsDTO(CrossReferenceFmsDTO dto) {
+		
+		ObjectResponse<List<CrossReference>> crResponse = new ObjectResponse<>();
+		List<CrossReference> xrefList = new ArrayList<>();
+		List<String> pageNames = List.of("default");
+		
+		if (CollectionUtils.isNotEmpty(dto.getPages())) {
+			pageNames = dto.getPages();
+		}
+		
+		for (String pageName : pageNames) {		
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not have trailing whitespace
<!-- __reviewdog__:ChBiMjFjZTk1YmIxMTEzNTgxEgpjaGVja3N0eWxl -->